actors/builtin/storage_miner/storage_miner_actor.go

```diff
@@ -511,7 +527,10 @@ func (a *StorageMinerActor) OnDeferredCronEvent(rt Runtime, params *OnDeferredCr
 	rt.ValidateImmediateCallerIs(builtin.StoragePowerActorAddr)
 
 	var payload CronEventPayload
-	serde.MustDeserialize(params.callbackPayload, payload)
+	err := payload.UnmarshalCBOR(bytes.NewReader(params.callbackPayload))
+	if err != nil {
+		rt.Abort(exitcode.ErrIllegalArgument, "failed to deserialize event payload")
```

Comment:
This abort message would be more helpful if it included the error. Can also save a line condensing the if statement `if err := ...; err != nil`